### PR TITLE
[DOCS] Add deletion sub feature privilege for cases

### PR DESCRIPTION
**Pull request by lcawl**
Relates to https://github.com/elastic/kibana/issues/136996

### Preview

https://security-docs_2219.docs-preview.app.elstc.co/guide/en/security/master/case-permissions.html

### CODE REVIEW

File: docs/getting-started/cases-req.asciidoc [hunk 1]
```diff
@@ -23,11 +23,14 @@ To grant access to cases, set the {kib} space privileges for the *Cases* and *Ac
...
 * `All` for the *Cases* feature
-* `All` for the *Action and Connector* feature (go to *Management > Actions and Connectors* to set this)
+* `All` for the *{connectors-feature}* feature (go to *Management > {connectors-feature}* to set this)
 
-NOTE: Roles without `All` *Actions and Connectors* feature privileges cannot create, add, delete, or modify case connectors.
+NOTE: Roles without `All` *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
 
 | Give view-only access for cases | `Read` for the *Cases* feature
+
+| Give access to view and delete cases | `Read` for the *Cases* feature with the deletion sub-feature enabled
```
Comment (nastasha-solomon) [inline docs/getting-started/cases-req.asciidoc:32]:
```suggestion
| Give access to view and delete cases | `Read` for the *Cases* feature and the *Delete* sub-feature selected
```

Comment (nastasha-solomon) [reply docs/getting-started/cases-req.asciidoc:32]: Mainly making this suggestion so users know exactly what to look for. Also changed "enabled" to "selected" to align the verb it with our [UI writing tips](https://wiki.elastic.co/pages/viewpage.action?pageId=105914797#Writingforandabouttheuserinterface(UI)-Choosingtherightword).

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for creating this @lcawl ! Do you think it'd be useful to expand the Cases privs to show the sub-feature privs? Or is it better to keep that area collapsed since we'll likely add to it in the future?

**Comment by lcawl** [comment]
> I do have one question about the screenshot refresh: do you think it'd be useful to expand the Cases privs to show the sub-feature privs?

I chose to leave it unexpanded since it's a subscription feature. However, if you disagree that's fine too. I also chose to not re-add the red boxes around the pertinent privileges, since we're generally trying to make screenshots that are easier to automate.  If you prefer to have those boxes here and don't mind manually maintaining, that's fine too.

**Review by benironside — APPROVED** [review]
LGTM. Fwiw I think it's fine not to include the red boxes in the screenshot. The image is described well enough in text that I think users will find the relevant permissions without them.

**Comment by nastasha-solomon** [comment]
@lcawl ah ok, good to know and I'm totally fine not annotating the screenshot. I feel like the scope of the screenshot is limited enough that users can quickly locate where the privs are when they reference the image.

**Pull request merged**